Documentation NFTS pallet

filippoweb3 opened the pull request:
This PRs aims to include in the wiki information about what it is possible to do with the new [nfts pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts).

Commit:
added links to substrate docs

Commit:
added link to nft page

Commit:
Edits to intro

Commit:
Added section nfts

- moved basilisk
- pallets at the end of the list

Commit:
Rearranged content

- kusama and then Polkadot parachains
- pallets first
- moved some content

Commit:
added nfts pallet functionalities

Commit:
Minor reorg

- distinction nft kusama and polkadot

DrW3RK committed:
Add a separate doc for NFT pallets

DrW3RK committed:
Move Uniques pallet to the pallets doc

profradha commented on docs/learn/learn-system-chains.md:
```diff
@@ -50,18 +50,17 @@ network, including amounts that have been transferred to other parachains. It is
-features.
+(NFTs) via the [Uniques pallet](https://polkadot.js.org/docs/substrate/extrinsics#uniques) and the
+new [nfts pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts). For mre information about
+NFTs see the [dedicated wiki page](./learn-nft.md).
```
```suggestion
(NFTs) via the [Uniques pallet](https://polkadot.js.org/docs/substrate/extrinsics#uniques) and the
new [nfts pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts). For more information about
NFTs see the [dedicated wiki page](./learn-nft-pallets.md).
```

DrW3RK reviewed (commented):
Made a couple of changes. The learn-nft page serves as a high level overview of NFTs and a brief into about the parachains/projects with NFT capabilities. Created learn-nft-pallets page to take a deep dive into the technical details of the NFT pallets.

filippoweb3 committed:
Update docs/learn/learn-system-chains.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

Commit:
Created NFT sidebar category

- put nft and nft-pallet pages there
- removed nft page from tokens and assets

Commit:
added links to nft pages

Commit:
Merge branch 'new-nfts-pallet' of https://github.com/w3f/polkadot-wiki into new-nfts-pallet

filippoweb3 commented:
@DrW3RK I created the "NFTs" category within "Basic" in the sidebar. Nfts are mentioned already in the "Asset" page within "Tokens and Assets". My point is that since we now have two pages related to NFTs (and I expect we will build more on them), NFTs deserve their category. It will also be easier to find them. Let me know what you think.

Commit:
Added short intro

Gravel927 commented on docs/learn/learn-nft.md:
```diff
@@ -1,14 +1,14 @@
 description: An explanation of the NFT landscape in the Polkadot ecosystem.
 keywords: [NFT, non-fungible token, NFT 2.0]
 slug: ../learn-nft
 ---
```
Drücken Sie lange auf ein Element in der Zwischenablage, um es anzupinnen. Nicht angepinnte Elemente werden nach einer Stunde gelöscht.

Commit:
added info nft fractionalization

Commit:
added PR link nft fract

Commit:
Minor edits, added links

Commit:
added things

- smart attributes
- mint settings
- feature flags
- multiple accounts per role

profradha commented on docs/learn/learn-assets.md:
```diff
@@ -133,10 +133,10 @@ stays up to date. For more info, see the "Moving Assets between Chains in XCM" s
+separate meaning from another instance of the same class.
 {{ polkadot: Statemint :polkadot }}{{ kusama: Statemine :kusama }} represents NFTs in the
-[Uniques pallet](https://crates.parity.io/pallet_uniques/index.html).
+[Uniques and Nfts pallets](./learn-nft-pallets.md).
```
```suggestion
[Uniques and NFTs pallets](./learn-nft-pallets.md).
```

profradha commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -0,0 +1,103 @@
+NFT [pallets](http://localhost:3000/docs/learn-extrinsics#pallets-and-extrinsics) allow substrate
+developers to easily implement NFT-related actions within their dApp.
+
+## NFTS Pallet
```
```suggestion
## NFTs Pallet
```

profradha commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -0,0 +1,103 @@
+---
+
+NFT [pallets](http://localhost:3000/docs/learn-extrinsics#pallets-and-extrinsics) allow substrate
+developers to easily implement NFT-related actions within their dApp.
```
```suggestion
NFT [pallets](learn-extrinsics.md#pallets-and-extrinsics) allow
developers to easily implement NFT-related actions within their dApp.
```

profradha commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -0,0 +1,103 @@
+### Current Implementations
+
+Nfts is a [FRAME pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts) currently deployed
+on Westmint that provides a multitude of functions to interact with NFTs.
```
```suggestion
NFTs is a [FRAME pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts) currently deployed
on Westmint which provides a multitude of functions to interact with NFTs.
```

profradha commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -0,0 +1,103 @@
+
+The pallet comes with a new way to configure it, as well as configure collections and items.
+Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow to disable
+functionalities that are not needed in the runtime.
```
By "it", do you mean "NFTs"?

```suggestion
The pallet comes with a new way to configure NFTs, as well as configure collections and items.
Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow disabling
functionalities that are not needed in the runtime.
```

profradha commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -0,0 +1,103 @@
+Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow to disable
+functionalities that are not needed in the runtime.
+
+There are features that common to the nft world such as:
```
```suggestion
Some features are common to the NFT world, such as:
```

profradha commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -0,0 +1,103 @@
+  involves setting some attributes specific to that item (an this is actually what drives the value
+  of the item). The creation of a collection also involves the specification of the maximum number
+  of items the collection has (an this also will drive the value of the collection's items in the
+  future).
```
Not all NFTs are meant for value creation. For instance, Soul bound NFTs, POAPs. 

```suggestion
- Buying an item up for sale.
- Creating (i.e., mint) and burning (i.e., destroy) a collection of items or a single item (burning must
  be signed either by the admin of the collection or the owner). Creating an item usually
    involves setting some attributes specific to that item. Creating a collection also requires the specification of the maximum number of items.
```

filippoweb3 committed:
Update docs/learn/learn-assets.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-nft-pallets.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-nft-pallets.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-nft-pallets.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-nft-pallets.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-nft-pallets.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-nft-pallets.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

Commit:
run grammarly

Commit:
run grammarly on learn-assets

Commit:
fixed broken links

Commit:
minor rearrangement of roles

Commit:
moved section

- creation and. management
- shared between fungible and non-fungible

Commit:
removed duplicated section about management

- minor edits, added links

Commit:
more links

- pure proxies & multisig

filippoweb3 commented:
@DrW3RK I added some minor edits to the "Asset" page. I think the next round of edits coming from the video we will watch can be added in another PR. Let me know what you think.

profradha commented on docs/learn/learn-assets.md:
```diff
@@ -57,26 +48,28 @@ required deposit of
-An asset class has a number of privileged roles. The creator of the asset automatically takes on all
+Asset classes and instances can have associated metadata. The metadata is an array of data that the
+class Owner can add on-chain, for example, a link to an IPFS hash or other off-chain hosting
+service. The [Uniques pallet](./learn-nft-pallets.md) also supports setting key/value pairs as
```
```suggestion
service. The [Uniques pallet](./learn-nft-pallets.md#uniques-pallet) also supports setting key/value pairs as
```

profradha commented on docs/learn/learn-assets.md:
```diff
@@ -57,26 +48,28 @@ required deposit of
 
-An asset class has a number of privileged roles. The creator of the asset automatically takes on all
+Asset classes and instances can have associated metadata. The metadata is an array of data that the
+class Owner can add on-chain, for example, a link to an IPFS hash or other off-chain hosting
```
```suggestion
class owner can add on-chain, for example, a link to an IPFS hash or other off-chain hosting
```

filippoweb3 committed:
Merge branch 'master' into new-nfts-pallet

DrW3RK reviewed (approved):
Looks good!

filippoweb3 committed:
Update docs/learn/learn-assets.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-assets.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

jsidorenko commented on docs/learn/learn-nft-pallets.md:
```diff
@@ -0,0 +1,101 @@
+
+[NFTs fractionalization](https://github.com/paritytech/substrate/pull/12565) will allow the user to:
+
+- Take ownership of an NFT from the [pallet-uniques](#uniques-pallet)
```
Should be: [pallet-nfts](#nfts-pallet)

profradha replied:
Updated! For the next PR, we will send it for your review before merging into the master.
